src/exampleTypes.js

```diff
@@ -263,6 +263,14 @@ export default F.stampKey('type', {
     },
   },
   pivot: {
+    validate: context =>
+      _.every(
+        ({ type, ranges, percents }) =>
+          (type !== 'numberRanges' && type !== 'percentiles') ||
+          (type === 'numberRanges' && ranges.length > 0) ||
+          (type === 'percentiles' && percents.length > 0),
```

Comment:
Do these throw an error? Maybe we should actually do this in contexture-es 🤔

Reply:
Yeah ES returns an error if it's empty

Reply:
OK, merging for now but we should fix this at the contexture-es level.

For a future revision, this kind of thing might be more cleanly modeled with an object lookup for type specific checks, e.g. something like this:

```js
let checks = {
  numberRanges: ({ ranges }) => ranges.length > 0,
  percentiles: ({ percents }) => percents.length > 0,
}
let checkGroup = group => _.getOr(_.stubTrue, group.type, checks)(group)
let validate = ({ groups }) => _.every(checkGroup, groups)
```